aws/data_source_aws_db_subnet_group.go

```diff
@@ -0,0 +1,93 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsDbSubnetGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsDbSubnetGroupRead,
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"subnets": {
```

Comment:
could we use "subnet_ids" here instead to match the schema field in the corresponding resource? will also help in testing as we can make use of `resource.TestCheckResourceAttrPair()`